app/src/androidTest/java/com/example/avjindersinghsekhon/minimaltodo/testApp/tests/Helpers.java

```diff
@@ -1,4 +1,129 @@
 
+import android.support.test.espresso.AppNotIdleException;
+import android.support.test.espresso.NoMatchingRootException;
+import android.support.test.espresso.NoMatchingViewException;
+import android.support.test.espresso.UiController;
+import android.support.test.espresso.ViewAction;
+import android.view.View;
+import android.widget.TextView;
+
+import junit.framework.AssertionFailedError;
+
+import org.hamcrest.Matcher;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.assertion.PositionAssertions.isAbove;
+import static android.support.test.espresso.assertion.PositionAssertions.isBelow;
+import static android.support.test.espresso.assertion.PositionAssertions.isLeftOf;
+import static android.support.test.espresso.assertion.PositionAssertions.isRightOf;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isAssignableFrom;
+import static android.support.test.espresso.matcher.ViewMatchers.isClickable;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+
 public class Helpers {
-}
+
+    public static boolean checkIfUIObjectIsVisible(Matcher<View> matcher) {
+        try {
+            onView(matcher).check(matches(isDisplayed()));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static boolean checkIfUIObjectIsClickable(Matcher<View> matcher) {
+        try {
+            onView(matcher).check(matches(isClickable()));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static boolean isUIObjectAboveAnotherObject(Matcher<View> matcher, Matcher<View> matcher2) {
+        try {
+            onView(matcher).check(isAbove(matcher2));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static boolean isUIObjectBelowAnotherObject(Matcher<View> matcher, Matcher<View> matcher2) {
+        try {
+            onView(matcher).check(isBelow(matcher2));
+            return true;
+        } catch (NoMatchingViewException | AppNotIdleException | AssertionFailedError | NoMatchingRootException e) {
+            return false;
+        }
+    }
+
+    public static boolean isRemindLeftOfSwitchButton(Matcher<View> matcher, Matcher<View> matcher2) {
```

Comment:
These are general objects. We have parameters :D

Reply:
Done.